wgpu-core/src/present.rs

```diff
@@ -225,7 +228,7 @@ impl Global {
                         mips: 0..1,
                     },
                     info: ResourceInfo::new(
-                        "<Surface Texture>",
+                        &Some(Cow::Borrowed("<Surface Texture>")),
```

Comment:
question(non-blocking): Should we expose an API for labeling surface textures? 🤔

Reply:
I tried doing that but things got quite involved since the API has to be compatible with both the `wgpu-core` and `webgpu` backends of `wgpu`. On the web there is currently no way to label a canvas. I think we could add a label field to `.configure()` but we'd need to spec that first.